src/main/java/com/faforever/api/data/domain/Avatar.java

```diff
@@ -22,14 +23,19 @@
 
   private String url;
   private String tooltip;
+  private String filename;
   private List<AvatarAssignment> assignments;
 
-  @Column(name = "url")
+  @Transient
   @NotNull
   public String getUrl() {
```

Comment:
I think this should be made @ComputedAttribute and being enriched by EntityLister. Take a look at Clan class and ClanEnricherListener